.github/workflows/test.yml

```diff
@@ -65,3 +65,4 @@ jobs:
         if: env.GIT_DIFF
         run: |
           make test-e2e
+          make test-integration
```

Comment:
nit: would create a separate job for this (unless if we plan on dep e2e entirely.

Reply:
Yep, that's the plan!